components/core/footer/Footer.vue

```diff
@@ -1,52 +1,70 @@
-                <locale-link to="/about/code-of-conduct">
-                    {{ $t('codeOfConduct') }}
-                </locale-link>
-                <!-- 
-                <locale-link to="/about/staff">{{ $t('staff') }}</locale-link>
-                <locale-link to="/about/community">
-                    {{ $t('community') }}
-                </locale-link>
-                <locale-link to="/about/privacy-policy">
-                    {{ $t('privacyPolicy') }}
-                </locale-link>
-                -->
+    <footer
+        class="w-full flex flex-col justify-between bg-dark-secondary text-golden"
+    >
+        <div class="w-full p-10">
+            <div class="h-32 flex justify-center">
+                <div class="flex flex-col">
+                    <locale-link to="/about/code-of-conduct">
+                        {{ $t('codeOfConduct') }}
+                    </locale-link>
+                    <!--
+                    <locale-link to="/about/staff">{{ $t('staff') }}</locale-link>
+                    <locale-link to="/about/community">
+                        {{ $t('community') }}
+                    </locale-link>
+                    <locale-link to="/about/privacy-policy">
+                        {{ $t('privacyPolicy') }}
+                    </locale-link>
+                    -->
+                </div>
             </div>
-            <div class="flex flex-col ml-20">
-                <ext-link href="https://pycontw.blogspot.tw/">
-                    Blogspot
-                </ext-link>
-                <ext-link href="https://www.facebook.com/pycontw">
-                    Facebook
-                </ext-link>
-                <ext-link href="https://twitter.com/pycontw">Twitter</ext-link>
-                <a
+            <div class="w-full flex justify-center items-center flex-wrap">
+                <footer-icon
+                    href="https://pycontw.blogspot.tw/"
+                    :icon="['fab', 'blogger']"
+                />
+                <footer-icon
+                    href="https://www/facebook.com/pycontw"
+                    :icon="['fab', 'facebook']"
+                />
+                <footer-icon href="" :icon="['fab', 'telegram']" />
+                <footer-icon
+                    href="https://twitter.com/pycontw"
+                    :icon="['fab', 'twitter-square']"
+                ></footer-icon>
+                <footer-icon
+                    href="https://www.youtube.com/PyConTaiwan"
+                    :icon="['fab', 'youtube-square']"
+                />
+                <footer-icon
                     href="mailto:organizers@pycon.tw"
-                    target="_blank"
-                    class="outline-none"
-                >
-                    Email
-                </a>
+                    :icon="['fas', 'envelope-square']"
+                />
+                <footer-icon
+                    href="https://tw.pycon.org/2020/zh-hant/ext/discord/"
```

Comment:
Let's replace the URL of the old instruction page with the invitation link of the `Python Taiwan` discord server. 

```suggestion
                    href="https://discord.com/invite/94hgCQv"
```

Reply:
updated